packages/cookie-consent/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@scaleway/cookie-consent",
+  "version": "0.2.14",
```

Comment:
Question: why using with this value, and not starting with 0.1 or similar?

Reply:
Copy paste issue, I am on it ;)